### PR TITLE
Experience/6311 [Message ID][Search][UI] View Results

### PR DESCRIPTION
**Pull request by penny-lischer**
This PR ...

To View:
1. Add message-id-search to the Feature Flags feature

## Changes
- Added Message Id Search page to the Admin dropdown
Screen Shot 2022-10-06 at 10 36 40 AM

Screen Shot 2022-10-06 at 10 36 50 AM

### CODE REVIEW

File: frontend-react/src/components/MessageIdSearch/MessageIdSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            {
+                dataAttr: "messageId",
+                columnHeader: "Message Id",
+            },
+            {
+                dataAttr: "sender",
+                columnHeader: "Sender",
+            },
+            {
+                dataAttr: "submittedAt",
```
Comment (ahay-agile6) [inline frontend-react/src/components/MessageIdSearch/MessageIdSearch.tsx:59]: according to `MOCK_MESSAGE_ID_DATA`, should this be `submittedDate`?

File: frontend-react/src/components/MessageIdSearch/MessageIdSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        id: 3,
+        messageId: "12-234567",
+        sender: "somebody 3",
+        submittedDate: "09/30/2022",
+        reportId: "26a37945-ed12-4578-b4f6-203e8b9d62ce",
+    },
+];
+
+// TODO: move this interface into the resources directory
+interface MessageIdResource {
```
Comment (ahay-agile6) [inline frontend-react/src/components/MessageIdSearch/MessageIdSearch.tsx:32]: should we call this `MessageSeachResource`? or maybe `MessageListResource`? `MessageSeachResultResource`? i feel having the word `Id` makes it confusing as the table is a list of Messages (not just ids)

**Review by ahay-agile6 — COMMENTED** [review]
looking good, just a couple of questions

File: frontend-react/src/components/MessageIdSearch/MessageIdSearch.tsx (hunk 1 above)
Comment (penny-lischer) [reply frontend-react/src/components/MessageIdSearch/MessageIdSearch.tsx:59]: Yes!

File: frontend-react/src/components/MessageIdSearch/MessageIdSearch.tsx (hunk 2 above)
Comment (penny-lischer) [reply frontend-react/src/components/MessageIdSearch/MessageIdSearch.tsx:32]: Will update

**Review by ahay-agile6 — APPROVED** [review]
lgtm. great work, @penny-lischer !!! thank you!

**Pull request merged**